js-miniapp-sdk/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "js-miniapp-sdk",
+  "version": "0.1.0",
+  "description": "Mini App SDK for JavaScript",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "dependencies": {},
```

Comment:
How about adding?
```
"dependencies": {
    "cross-env": "5.0.5"
  }
```

Reply:
I think that was just an example on the Yarn documentation, so we don't need it. We can add it in the future if we need to set any environment variables.